src/LibraryViewExtensionWebView2/ViewExtension.cs

```diff
@@ -37,7 +38,17 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
                 controller = new LibraryViewController(viewLoadedParams.DynamoWindow, viewLoadedParams.CommandExecutive, customization);
                 (viewLoadedParams.DynamoWindow.DataContext as DynamoViewModel).PropertyChanged += handleDynamoViewPropertyChanges;
             }
-            
+            viewParams.CurrentWorkspaceChanged += ViewParams_CurrentWorkspaceChanged;
+        }
+
+        private void ViewParams_CurrentWorkspaceChanged(IWorkspaceModel workspace)
+        {
+            var type = workspace switch
+            {
+                HomeWorkspaceModel _ => "home",
```

Comment:
what do you think of having an enum instead of a hard-coded string?

Reply:
should be straight forward, will look into it.